package configuration for libffi
cflags: -I/opt/local/lib/libffi-3.0.9/include  
ldflags: -L/opt/local/lib
libs: -lffi  

have_library: checking for ffi_call() in -lffi... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib     -lffi   -lruby.1.9.1-static  -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: int main() {return 0;}
/* end */

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib     -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: #include <ffi.h>
4: 
5: /*top*/
6: int main() {return 0;}
7: int t() { void ((*volatile p)()); p = (void ((*)()))ffi_call; return 0; }
/* end */

--------------------

have_func: checking for ffi_prep_closure()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
conftest.c: In function ‘t’:
conftest.c:5: error: ‘ffi_prep_closure’ undeclared (first use in this function)
conftest.c:5: error: (Each undeclared identifier is reported only once
conftest.c:5: error: for each function it appears in.)
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))ffi_prep_closure; return 0; }
/* end */

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { ffi_prep_closure(); return 0; }
/* end */

--------------------

have_func: checking for ffi_raw_call()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
conftest.c: In function ‘t’:
conftest.c:5: error: ‘ffi_raw_call’ undeclared (first use in this function)
conftest.c:5: error: (Each undeclared identifier is reported only once
conftest.c:5: error: for each function it appears in.)
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))ffi_raw_call; return 0; }
/* end */

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { ffi_raw_call(); return 0; }
/* end */

--------------------

have_func: checking for ffi_prep_raw_closure()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
conftest.c: In function ‘t’:
conftest.c:5: error: ‘ffi_prep_raw_closure’ undeclared (first use in this function)
conftest.c:5: error: (Each undeclared identifier is reported only once
conftest.c:5: error: for each function it appears in.)
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))ffi_prep_raw_closure; return 0; }
/* end */

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { ffi_prep_raw_closure(); return 0; }
/* end */

--------------------

have_func: checking for rb_thread_blocking_region()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))rb_thread_blocking_region; return 0; }
/* end */

--------------------

have_func: checking for ruby_thread_has_gvl_p()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
conftest.c: In function ‘t’:
conftest.c:5: error: ‘ruby_thread_has_gvl_p’ undeclared (first use in this function)
conftest.c:5: error: (Each undeclared identifier is reported only once
conftest.c:5: error: for each function it appears in.)
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))ruby_thread_has_gvl_p; return 0; }
/* end */

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { ruby_thread_has_gvl_p(); return 0; }
/* end */

--------------------

have_func: checking for ruby_native_thread_p()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))ruby_native_thread_p; return 0; }
/* end */

--------------------

have_func: checking for rb_thread_call_with_gvl()... -------------------- yes

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
conftest.c: In function ‘t’:
conftest.c:5: error: ‘rb_thread_call_with_gvl’ undeclared (first use in this function)
conftest.c:5: error: (Each undeclared identifier is reported only once
conftest.c:5: error: for each function it appears in.)
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { void ((*volatile p)()); p = (void ((*)()))rb_thread_call_with_gvl; return 0; }
/* end */

"/usr/bin/gcc-4.2 -o conftest -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/x86_64-darwin11.0.0 -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1/ruby/backward -I/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/include/ruby-1.9.1 -I. -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE   -isysroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -fno-common -pipe -I/opt/local/lib/libffi-3.0.9/include   conftest.c  -L. -L/Users/aldo/.rvm/rubies/ruby-1.9.2-p290/lib -L. -Wl,-syslibroot /Developer/SDKs/MacOSX10.7.sdk -arch x86_64 -L/usr/local/lib -L/opt/local/lib    -lffi  -lffi   -lruby.1.9.1-static -lffi  -lffi   -lpthread -ldl -lobjc "
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: /*top*/
4: int main() {return 0;}
5: int t() { rb_thread_call_with_gvl(); return 0; }
/* end */

--------------------

